praas-app-ionic/src/pages/Home/index.tsx

```diff
@@ -1,17 +1,14 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import { IonPage, IonContent } from '@ionic/react';
 import { connect } from 'react-redux';
-import { withRouter, RouteComponentProps } from 'react-router';
+import { RouteComponentProps } from 'react-router';
```

Comment:
I assume we are not relying on RouteComponentProps here too

Reply:
Removed since we are not using `history` or `location` here